src/test/java/esa/sen2vm/Sen2VMTest.java

```diff
@@ -13,7 +13,7 @@
 /**
  * Unit test for simple App.
  */
-public class AppTest
+public class Sen2VMTest
 {
     /**
      * Rigorous Test :-)
```

Comment:
#smiley choqué

Reply:
I push an unit test that read a configuration file and print some of its content: https://github.com/sen2vm/sen2vm-core/pull/7/commits/f518545c9c99d14d39b5ed4131a13490b8f1a83a